root/static/scripts/common/components/EntityLink.js

```diff
@@ -19,7 +19,10 @@ import isolateText from '../utility/isolateText';
 import nonEmpty from '../utility/nonEmpty';
 import reactTextContent from '../utility/reactTextContent';
 
-export const DeletedLink = ({name, allowNew}: {|+name: React.Node, +allowNew: boolean|}) => {
+export const DeletedLink = ({
+  name,
+  allowNew,
```

Comment:
I'm surprised it doesn't want these in alphabetical order too. Should change it for consistency I guess?

Reply:
Changes made